### PR TITLE
Prevent department_admin of being displayed as a component.

### PR DESCRIPTION
**Pull request by tramuntanal**
#### :tophat: What? Why?
Prevent department_admin of being displayed as a component.

#### :pushpin: Related Issues
- Related to #?
- Fixes #?

#### :clipboard: Subtasks
- [x] Add `CHANGELOG` entry
- [-] Add documentation regarding the feature 
- [-] Add/modify seeds
- [-] Add tests
- [x] Depend upon department_admin v0.0.2.

#### Sreenshots
imatge

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,12 +1,13 @@
 source "https://rubygems.org"
 
 ruby RUBY_VERSION
-DECIDIM_VERSION = { git: 'https://github.com/gencat/decidim', tag: 'v0.18-dev-gencat-features' }.freeze
+# DECIDIM_VERSION = { git: 'https://github.com/gencat/decidim', tag: 'v0.18-dev-gencat-features' }.freeze
+DECIDIM_VERSION = { path: '../decidim-fork' }.freeze
```
Comment (aitorlb) [inline Gemfile]: This is intentional?

Comment (isaacmg410) [reply Gemfile]: Iep! @tramuntanal

Comment (tramuntanal) [reply Gemfile]: I may have introduced it by error when merging a stash. @isaacmg410 can you please confirm?

Comment (tramuntanal) [reply Gemfile]: Thanks @aitorlb you were right! fixed!

**Pull request merged**